src/command_modules/azure-cli-cosmosdb/HISTORY.rst

```diff
@@ -12,8 +12,15 @@ Release History
 ++++++++++++++++++
 * Minor fixes.
 
+0.1.6+dev
+++++++++++++++++++
+
+* Added Support for Collection Default TTL.
+* pyproj files updated to include cosmosdb files.
```

Comment:
thanks, addressed in my most recent commit:
069e6d0ea50b2270ef8d1cee599e98934f5113b7